website/docs/cli/private_registry/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+description: >-
+  Documentation listing third party projects that implement private registries 
+  and their corresponding guides for setting them up.
+---
+
+# Private Registries
+
+OpenTofu, by default, uses the public registry at [registry.opentofu.org](https://opentofu.org/registry/). 
+However, organizations may have a need for a private registry for modules, providers, or policies that need to be kept private.
+
+There are several project that implement the [terraform registry API](https://developer.hashicorp.com/terraform/registry/api-docs), 
+aimed at providing this functionality of hosting a private registry. They are also
+compatible with OpenTofu.
+
+### List of Private Registries
+
+| Registry Name                     | Documentation                                     |
+| ----------------------------      | ------------------------------------------------------ |
+| Gitlab                            | [Terraform Module Registry](https://docs.gitlab.com/ee/user/packages/terraform_module_registry/)
+| Citizen                           | [outsideris/citizen](https://github.com/outsideris/citizen)                   |
+| Anthology                         | [erikvanbrakel/anthology](https://github.com/erikvanbrakel/anthology)         |
+| Boring Registry                   | [boring-registry/boring-registry](https://github.com/boring-registry/boring-registry) |
+| Terraform Registry by NRK         | [nrkno/terraform-registry](https://github.com/nrkno/terraform-registry)        |
```

Comment:
Why are there some mentioned by vendor, others not? I'd try to keep it consistent.

Reply:
Some have names given to their registry implementation, but some are named generically like the ones from nrk and philips labs. Should I name all of them by the vendor? because that will certainly be different for all.

Reply:
Also, many vendors are just individuals